docs/cheatsheet/repl.rst

```diff
@@ -0,0 +1,46 @@
+.. _ref_cheatsheet_repl:
+
+REPL
+====
+
+List databases:
```

Comment:
The most important command `\d` is missing. Give a couple examples

Reply:
\d std::Object
\d decimal
\d sum

Reply:
OK